pvmismatch/pvmismatch_lib/pvconstants.py

```diff
@@ -111,6 +111,8 @@ def npts(self, npts):
         # so that tight spacing is around MPP and RBD
```

Comment:
I made a custom version of plotSys() and ran:
```
pvconst = pvconstants.PVconstants(npts=20) # I did this just to see the spacing better
pvsys = pvsystem.PVsystem(pvconst=pvconst)
f = pvsys.plotSys()
f.show()
```
![spacing](https://user-images.githubusercontent.com/30896066/65149444-633a0200-da22-11e9-93fa-caccbbd2df49.png)
Maybe I'm misunderstanding something, but I was wondering if this is what you wanted to achieve. Now MPP falls into the non-tightly spaced part of the second section of sample points and it gets more tightly spaced towards VOC. Shouldn't the second section be reversed?

Reply:
Thanks. I see your point! But I'm ambivalent. When calculating current near Voc, very small changes in voltage result in big changes in current, and that region near Voc has a smaller voltage range than the Isc side of the mpp, but same `npts`, so it's already dense. So I'm not sure what's better. @chetan201? I think you might be right, but I'll ask a few others.

Reply:
fixed in 95644c2df334c8ee31588a217ac4875ea8162682